packages/lesmis-server/src/connections/SessionHandlingApiConnection.ts

```diff
@@ -3,7 +3,8 @@
  * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
  */
 
-import { ApiConnection, AuthHandler, SessionType } from '@tupaia/server-boilerplate';
+import { ApiConnection, AuthHandler } from '@beyondessential/tupaia-api-client';
```

Comment:
As above

>I decided to move all of the api-client logic out of server-boilerplate so that devs wouldn't add more code there. So we have some quite unsightly imports used in deprecated classes.